src/datumaro/cli/commands/prune.py

```diff
@@ -0,0 +1,139 @@
+        When not specified, the current project's working tree is used.|n
+        |n
+        By default, datasets are updated in-place. The '-o/--output-dir'
+        option can be used to specify another output directory. When
+        updating in-place, use the '--overwrite' parameter (in-place
+        updates fail by default to prevent data loss), unless a project
+        target is modified.|n
+        |n
+        The command can be applied to a dataset or a project build target,
+        a stage or the combined 'project' target, in which case all the
+        targets will be affected.|n
+        |n
+        Examples:|n
+        - Prune dataset with selecting random and ratio 80%:|n
+        |s|s%(prog)s -m random -r 0.8|n
+        - Prune dataset with clustering in image hash and ratio 50%:|n
+        |s|s%(prog)s -m query_clust -h img -r 0.5|
+        - Prune dataset based on entropy with clustering in image hash and ratio 50%:|n
+        |s|s%(prog)s -m entropy -h img -r 0.5|
+        """,
+        formatter_class=MultilineFormatter,
+    )
+
+    parser.add_argument("target", nargs="?", help="Target dataset revpath (default: project)")
+    parser.add_argument("-m", "--method", dest="method", help="Method to apply to the dataset")
+    parser.add_argument(
+        "-r", "--ratio", type=float, dest="ratio", help="How much to remain dataset after pruning"
+    )
+    parser.add_argument(
+        "--hash-type",
+        type=str,
+        dest="hash_type",
+        default="img",
+        help="Hashtype to extract feature from data information between image and text(label)",
+    )
+    parser.add_argument(
+        "-p",
+        "--project",
+        dest="project_dir",
+        help="Directory of the project to operate on (default: current dir)",
+    )
+
+    parser.add_argument(
+        "-o",
+        "--output-dir",
+        dest="dst_dir",
+        help="""
+            Output directory. Can be omitted for main project targets
+            (i.e. data sources and the 'project' target, but not
+            intermediate stages) and dataset targets.
+            If not specified, the results will be saved inplace.
+            """,
+    )
+    parser.add_argument(
+        "--overwrite", action="store_true", help="Overwrite existing files in the save directory"
+    )
+    parser.add_argument(
+        "--model", dest="model", default="clip", help="Model to use for hash inference"
+    )
+    parser.add_argument(
+        "--stage",
+        type=str_to_bool,
+        default=True,
+        help="""
+            Include this action as a project build step.
+            If true, this operation will be saved in the project
+            build tree, allowing to reproduce the resulting dataset later.
+            Applicable only to main project targets (i.e. data sources
+            and the 'project' target, but not intermediate stages)
+            (default: %(default)s)
+            """,
+    )
```

Comment:
Are those things being used for now?

Reply:
Remove this. For model, it will be updated later PR.